docs/nginx-setup.md

```diff
@@ -0,0 +1,74 @@
+# Setting up LetsEncrypt Certificate with Nginx Reverse Proxy
+
+- setup virtual host config as in `./nginx.conf`
+
+- create a softlink
+```
+  sudo ln -s /etc/nginx/sites-available/subsites /etc/nginx/sites-enabled/subsites
+```
+
+- get a cert using LetsEncrypt (Make sure LetsEncrypt is installed)
```

Comment:
Looks really good! Can you please add a link to instructions for installing Let's Encrypt? I know that there are various different NodeJS and platform specific packages/libs.